js/script.js

```diff
@@ -0,0 +1,36 @@
+const taskInput = document.getElementById("task-input");
+const addButton = document.getElementById("btn-add");
+const unoderedList = document.getElementById("tasks");
+const notification = document.getElementById("message");
+
+let tasks = [];
+const addTaskHandler = () => {
+  const taskTitle = taskInput.value.trim();
+  if (taskTitle) {
+    tasks.unshift(taskTitle);
+    const tasksList = document.createElement("li");
+    tasksList.textContent = taskTitle;
+    unoderedList.prepend(tasksList);
+    taskInput.value = "";
+    showMessage("Task added successfully!");
+    addButton.disabled = true;
+  }
+};
+
+const showMessage = (message) => {
+  notification.textContent = message;
+  notification.hidden = false;
+  setTimeout(() => {
+    notification.hidden = true;
+  }, 1000);
+};
+
+addButton.addEventListener("click", addTaskHandler);
```

Comment:
Since we are only using one `eventLister` (click in this case) for the add button why aren't we using 

```
addButton.onclick = () => {
    // do stuff here
};

```

May I know your thoughts on this?

Reply:
Bhai, there is no specific reason for using the `addEventListener()` approach, can use the `onclick` approach also. But because `addEventListener()` approach is flexible, more maintainable _(allows multiple event listeners to be added or removed as needed)_   whenever multiple events are used, that's why thought to go with it. Although here, it is mono-event function `onclick` approach can be a better choice.

Thank you

Reply:
Let's then refactor the code using `.onClick`?